source/style-guide/style/titles-and-headings.txt

```diff
@@ -134,70 +112,119 @@ guides, and tables, figures, and examples follow this list.
 
-         MongoDB Auto Scale glossary
-       - Environment variables for the nova and supernova clients
+         MongoDB Auto Scale Glossary
+       - Environment Variables for the Nova and Supernova Clients
 
-         Restore operations
+         Restore Operations
 
-         cURL command summary
+         cURL Command Summary
 
      * - Troubleshooting
        - A grammatical structure that's appropriate for the type of
          content (a troubleshooting topic can contain task, tutorial,
          concept, or reference information)
-       - Troubleshoot alarms
+       - Troubleshoot Alarms
 
-         Service troubleshooting on Linux
+         Service Troubleshooting on Linux
        - Troubleshooting
 
      * - FAQ
        - A descriptive noun or noun phrase, followed by *FAQ*
        - MongoDB Cloud Billing FAQ
 
-         Scheduled images FAQ
+         Scheduled Images FAQ
        - Not applicable
 
+SEO Guidelines for Titles
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To learn more about SEO guidelines for titles, see the 
+:ref:`Titles <seo-guidelines-titles>` section of Search Engine Optimization Guidelines.
+
+General Style and Structure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following guidelines apply to all titles and headings:
+
+- Create succinct, meaningful, descriptive titles and headings, and
+  place the most important words first.
+
+- Ensure that each title and heading is unique. Identical titles, even 
+  between documentation sets, might compete in search results.
+
+- Don't include "MongoDB" in a title unless the page is a product 
+  landing page.
+
+- Include articles, prepositions, and punctuation as needed for
+  clarity. However, avoid using an article (*a*, *an*, or *the*) as the
+  first word.
+
+- Avoid showing both an abbreviation and its spelled-out term in a
+  title or heading. To help control the length of titles and headings,
+  show the abbreviation in the title or heading and then define it in
+  the first paragraph of the text.
+
+- If you show a literal term (such as a command or option name) in a
+  title or heading, follow it with an appropriate noun.
+
+- Don't end a title or heading with a colon or period. If the title or
+  heading is in the form of a question, end it with a question mark.
+
+- Don't apply font treatments (bold, italics, or monospace) to text in
+  a title or heading.
+
+- Don't include trademark symbols in titles or headings. Show the
+  symbol on the first use of the trademark in text.
+
+- Avoid having only a single heading at any level (for example, a
+  single subsection in an article or section). If you find that you
+  have a single heading at any one level, consider whether you can
+  reorganize the information to either eliminate the heading or add a
+  second one at that level.
+
+- Avoid having more than two levels of sections within an article or
+  topic. If you use more than two levels of sections, consider whether
+  you can reorganize to make the structure flatter.
```

Comment:
I'm not sure I understand what this means. Could you share an example?

Reply:
Changed sections to say 'headings' instead. Hopefully that makes more sense!